CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ find_package(CURL REQUIRED)
 find_package(DirectX)
 find_package(epoxy)
 find_package(expat REQUIRED)
-find_package(freetype)
+find_package(freetype REQUIRED)
```

Comment:
Can you update `cmake/Dependencies.cmake` (around line 45) to mark this as required there too?

Reply:
Bump